### ISSUE TITLE
Merge DedupliFHIR Update PRs

### ISSUE THREAD

**Issue by IsaacMilarky** [DSACMS/dedupliFHIR#94]
**Is your feature request related to a problem? Please describe.**
Currently, there are a number of Dependabot update requests pending on the DedupliFHIR Repository

**Describe the solution you'd like**
The pending dependabot PRs should be tested and then merged into dev unless they have a security alert associated with them and in that case they should be merged directly into main. 

**Additional context**
There are already frontend and backend tests in the checks.yml file that should test these branches for you but you should still evaluate the changes based on your own expertise before approving these PRs

**Comment by decause-gov** [comment]
- Talk with Ricardo about how this works. Murt will be back next week. Primary focus should be on Demos/Decks, but you might be able to sneak this one in.
